### PR TITLE
Search page

### PR DESCRIPTION
**Pull request by Efekahya**
resim

Figma:
<a href="https://www.figma.com/file/SBPCNjmePIwJCzqqhXF9Y8/Rick-and-Morty---Digieggs-Assessment?node-id=1016%3A8998">Search Bar</a>
<a href="https://www.figma.com/file/SBPCNjmePIwJCzqqhXF9Y8/Rick-and-Morty---Digieggs-Assessment?node-id=1032%3A5997">Search Results</a>

### CODE REVIEW

File: src/components/Navbar/index.tsx [hunk 1]
```diff
@@ -32,18 +53,112 @@ export default function Navbar({ Logo }: INavbarProps) {
...
+      //           className="navbar-dropdown-item"
+      //           onClick={() => {
+      //             setSearchValue(name);
+      //             setSearchResultsNames([]);
+      //           }}
+      //         >
+      //           {name}
+      //         </button>
+      //       ])
+      //     );
```
Comment (atvdi) [inline src/components/Navbar/index.tsx]: Please remove unnecessary codes

File: src/components/Navbar/index.tsx [hunk 2]
```diff
@@ -32,18 +53,112 @@ export default function Navbar({ Logo }: INavbarProps) {
...
+              <>
+                <div className="navbar-episodes-item">
+                  <EpisodeCard
+                    key={id}
+                    title={name}
+                    date={air_date}
+                    episode={episode}
+                    description="lorem ipsum"
+                    favorited={false}
+                    handleSetFavorited={() => console.log("here")}
```
Comment (atvdi) [inline src/components/Navbar/index.tsx:110]: Remove console logs

File: src/components/Navbar/index.tsx [hunk 3]
```diff
@@ -32,18 +53,112 @@ export default function Navbar({ Logo }: INavbarProps) {
...
 
+  if (loading === false && data) {
+    window.onscroll = e => {
+      console.log("scrolling");
+      e.preventDefault();
+      if (
+        window.innerHeight + document.documentElement.scrollTop + 40 >
+        document.documentElement.offsetHeight
+      ) {
+        console.log("bottom");
```
Comment (atvdi) [inline src/components/Navbar/index.tsx]: console log

**Review by atvdi — CHANGES_REQUESTED** [review]

File: src/components/Navbar/index.tsx (hunk 2 above)
Comment (Efekahya) [reply src/components/Navbar/index.tsx:110]: this will be changed when favoriteContext is introduced

**Pull request merged**